### DISCUSSION TITLE
Coverage Gutters guide as a new user

### DISCUSSION THREAD

**Discussion by ghost** [ryanluker/vscode-coverage-gutters#316] [Q&A]
Hi, I am currently trying to use Coverage Gutters and the end result does not seems what I want it to be, maybe I did something wrong or the configuration of the settings are incorrect.

This is what I did from the code downloaded at the examples side. First I added some new lines of code into the file, 'test_sample.py', as shown in the picture. New lines of code are added in test_2:

![1](https://user-images.githubusercontent.com/82016795/120177347-a34a1400-c208-11eb-8753-8e0036a838f4.png)

After that, the codes are then debug and go through the testing phase, which will then shown as in the following picture:

![2](https://user-images.githubusercontent.com/82016795/120177693-f7ed8f00-c208-11eb-88d0-0a308059474f.png)

Lastly the 'Watch' button is being clicked but the test_2 section is not covered with red lines:

![3](https://user-images.githubusercontent.com/82016795/120177813-1784b780-c209-11eb-9ac2-29c97f42ac22.png)

So I would like to ask do I need to change any of the settings in the Visual code to do it? Or do I do anything wrongly? Because this is my first time doing it and I tried some research on the internet and to no avail, I could not find any information about it. A ZIP file of the code are added at the bottom. Thank you for your help and guidance.

[vscode-coverage-gutters-master (1).zip](https://github.com/ryanluker/vscode-coverage-gutters/files/6569454/vscode-coverage-gutters-master.1.zip)

**Comment by ryanluker** [comment]
@maxcwy2018 Thanks for submitting a discussion item, most likely you just need to rerun the coverage generation to get the new lines to show up. You can find this command in the readme for that example python project https://github.com/ryanluker/vscode-coverage-gutters/tree/master/example/python .
